pkg/blockdevice/lvm/logical_volumes.go

```diff
@@ -187,7 +187,7 @@ func (lv *LogicalVolume) Activate(ctx context.Context, rawArgs string) (bool, er
 		}
 	}
 
-	return false, fmt.Errorf("could not find logical volume %s: %w", lv.Name, err)
+	return false, fmt.Errorf("could not find logical volume %s", lv.Name)
```

Comment:
Why drop the error detail here?

Reply:
`err` is nil here, so it was a mistake to include it originally. That's why we would see this error message:
"could not find logical volume lv-0: %!w(<nil>)"